<?xml version="1.0" encoding="utf-8"?>
<TestRun id="18b56c0f-e60a-4987-835b-273b14c06e87" name="brand@BRANDONWINDOWS 2025-09-17 18:24:38" runUser="BRANDONWINDOWS\brand" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2025-09-17T18:24:38.3009697+02:00" queuing="2025-09-17T18:24:38.3009699+02:00" start="2025-09-17T18:24:34.7336787+02:00" finish="2025-09-17T18:24:38.3081399+02:00" />
  <TestSettings name="default" id="1da0a909-5f16-464f-a9ae-c415a6f92cd7">
    <Deployment runDeploymentRoot="brand_BRANDONWINDOWS_2025-09-17_18_24_38" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="c1d14dc3-768f-42b7-b7d8-ffbff047fda8" testId="42bc6be1-e8dc-93fe-066b-cc0dd7a94799" testName="AuthenticationAPI.IntegrationTests.AdminInviteActivateTests.Admin_Invites_User_User_Activates_With_Roles" computerName="BRANDONWINDOWS" duration="00:00:02.9447978" startTime="2025-09-17T18:24:38.1791831+02:00" endTime="2025-09-17T18:24:38.1792141+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="c1d14dc3-768f-42b7-b7d8-ffbff047fda8">
      <Output>
        <ErrorInfo>
          <Message>Expected boolean to be true, but found False.</Message>
          <StackTrace>   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)&#xD;
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)&#xD;
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)&#xD;
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)&#xD;
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)&#xD;
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)&#xD;
   at FluentAssertions.Primitives.BooleanAssertions`1.BeTrue(String because, Object[] becauseArgs)&#xD;
   at AuthenticationAPI.IntegrationTests.AdminInviteActivateTests.CreateSeededAdminAsync() in C:\Users\brand\Desktop\BCMTrac Auth Service\AuthenticationAPI\AuthenticationAPI.IntegrationTests\AdminInviteActivateTests.cs:line 40&#xD;
   at AuthenticationAPI.IntegrationTests.AdminInviteActivateTests.Admin_Invites_User_User_Activates_With_Roles() in C:\Users\brand\Desktop\BCMTrac Auth Service\AuthenticationAPI\AuthenticationAPI.IntegrationTests\AdminInviteActivateTests.cs:line 54&#xD;
--- End of stack trace from previous location ---</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="AuthenticationAPI.IntegrationTests.AdminInviteActivateTests.Admin_Invites_User_User_Activates_With_Roles" storage="c:\users\brand\desktop\bcmtrac auth service\authenticationapi\authenticationapi.integrationtests\bin\debug\net8.0\authenticationapi.integrationtests.dll" id="42bc6be1-e8dc-93fe-066b-cc0dd7a94799">
      <Execution id="c1d14dc3-768f-42b7-b7d8-ffbff047fda8" />
      <TestMethod codeBase="C:\Users\brand\Desktop\BCMTrac Auth Service\AuthenticationAPI\AuthenticationAPI.IntegrationTests\bin\Debug\net8.0\AuthenticationAPI.IntegrationTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="AuthenticationAPI.IntegrationTests.AdminInviteActivateTests" name="Admin_Invites_User_User_Activates_With_Roles" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="42bc6be1-e8dc-93fe-066b-cc0dd7a94799" executionId="c1d14dc3-768f-42b7-b7d8-ffbff047fda8" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" passed="0" failed="1" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.8.2+699d445a1a (64-bit .NET 8.0.20)&#xD;
[xUnit.net 00:00:00.07]   Discovering: AuthenticationAPI.IntegrationTests&#xD;
[xUnit.net 00:00:00.10]   Discovered:  AuthenticationAPI.IntegrationTests&#xD;
[xUnit.net 00:00:00.10]   Starting:    AuthenticationAPI.IntegrationTests&#xD;
[xUnit.net 00:00:03.09]       Expected boolean to be true, but found False.&#xD;
[xUnit.net 00:00:03.09]       Stack Trace:&#xD;
[xUnit.net 00:00:03.09]            at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)&#xD;
[xUnit.net 00:00:03.09]            at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)&#xD;
[xUnit.net 00:00:03.09]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)&#xD;
[xUnit.net 00:00:03.09]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)&#xD;
[xUnit.net 00:00:03.09]            at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)&#xD;
[xUnit.net 00:00:03.09]            at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)&#xD;
[xUnit.net 00:00:03.09]            at FluentAssertions.Primitives.BooleanAssertions`1.BeTrue(String because, Object[] becauseArgs)&#xD;
[xUnit.net 00:00:03.09]         C:\Users\brand\Desktop\BCMTrac Auth Service\AuthenticationAPI\AuthenticationAPI.IntegrationTests\AdminInviteActivateTests.cs(40,0): at AuthenticationAPI.IntegrationTests.AdminInviteActivateTests.CreateSeededAdminAsync()&#xD;
[xUnit.net 00:00:03.09]         C:\Users\brand\Desktop\BCMTrac Auth Service\AuthenticationAPI\AuthenticationAPI.IntegrationTests\AdminInviteActivateTests.cs(54,0): at AuthenticationAPI.IntegrationTests.AdminInviteActivateTests.Admin_Invites_User_User_Activates_With_Roles()&#xD;
[xUnit.net 00:00:03.09]         --- End of stack trace from previous location ---&#xD;
[xUnit.net 00:00:03.12]   Finished:    AuthenticationAPI.IntegrationTests&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="BRANDONWINDOWS" outcome="Error" timestamp="2025-09-17T18:24:38.1766960+02:00">
        <Text>[xUnit.net 00:00:03.09]     AuthenticationAPI.IntegrationTests.AdminInviteActivateTests.Admin_Invites_User_User_Activates_With_Roles [FAIL]</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>